src/ResourceManager/Websites/Commands.Websites/Cmdlets/BackupRestore/AzureWebAppBackupConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+
+using System;
+using Microsoft.Azure.Management.WebSites.Models;
+
+namespace Microsoft.Azure.Commands.WebApps.Cmdlets.WebApps
+{
+    public class AzureWebAppBackupConfiguration
+    {
+        public string Name { get; set; }
```

Comment:
Since this is a type users might have to construct, documentation comments for the type and properties would be useful.